internal/middleware/auth.go

```diff
@@ -45,6 +46,13 @@ func Auth(params AuthParams) gin.HandlerFunc {
 			return
 		}
 
+		sentry.ConfigureScope(func(scope *sentry.Scope) {
+			scope.SetUser(sentry.User{
+				ID:    currentUser.ID.String(),
+				Email: currentUser.Email,
```

Comment:
Not sure, may pose some data privacy concerns.
Don't think we need to care about that for now, as we don't have user logins yet.